jmkarthik-kubectl_resource.yml

```diff
@@ -0,0 +1,6 @@
+name: kubectl-resource

```

Comment:
we intend to have the `name` reflect the name of the resource type that would be used in the pipeline. In this case I am assuming it would be just `kubectl`. If that is true, do you mind changing this to `kubectl`?

Reply:
@YoussB I have updated the name to kubectl.